tools/talis/genesis.go

```diff
@@ -146,6 +153,34 @@ func createPayload(ips []Instance, chainID, ppath string, squareSize int, mods .
 	return nil
 }
 
+// mainnetVotingPowers contains the current Celestia mainnet staking distribution for more realistic tests.
+var mainnetVotingPowers []int
```

Comment:
[question] why is this defined in global scope if it is only assigned to inside `getMainnetStake` and only used inside `getMainnetStake`? Can it be moved inside that function?